sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeCustomFormsOperation.cs

```diff
@@ -80,9 +87,15 @@ internal RecognizeCustomFormsOperation(ServiceClient operations, ClientDiagnosti
         /// <param name="client">The client used to check for completion.</param>
         public RecognizeCustomFormsOperation(string operationId, FormRecognizerClient client)
         {
-            Id = operationId;
             _serviceClient = client.ServiceClient;
             _diagnostics = client.Diagnostics;
+
+            string[] substrs = operationId.Split('/');
+
+            _resultId = substrs.Last();
+            _modelId = substrs.First();
+
+            Id = operationId;
```

Comment:
looks like Id is no longer used. Could we delete it?

Reply:
It can still be accessed by the user. It's a requirement for every `Operation` subclass.